Dockerfile

```diff
@@ -16,4 +16,4 @@ ENTRYPOINT ["entrypoint.sh"]
 
 EXPOSE 3013
 
-CMD ["rails", "server", "-p", "3013", "-b", "0.0.0.0"]
```

Comment:
we changed [puma.rb](https://github.com/clio/example-third-party-application/pull/33/files#diff-24409379bdb75ad446bb4e2c18fd4cced1a263b99a3ff96fc3777d8fd8faeab9R18) to use 3013 by default so we don't need to set the port manually anymore

Reply:
I see the change now. Thanks